addons/website/static/src/js/website.snippets.gallery.js

```diff
@@ -360,7 +362,7 @@ options.registry.gallery_img = options.Class.extend({
         if (type !== "click") return;
 
         var self = this;
-        var $parent = snippet_editor.globalSelector.closest(this.$target.parent());
+        var $parent = this.$target.closest('section');
```

Comment:
just remove parent(), globalSelector.closest return the closest parent snippet. (before the last changes, the image have snippet but now the image option is in the top editor bar)